developer_setup.md

```diff
@@ -43,32 +18,25 @@ As per 20+, with the following changes:
   sudo dnf -y install postgresql-devel postgresql-server # PostgreSQL Database server and to build 'pg' Gem
   sudo dnf -y install libxml2-devel libxslt-devel patch  # For Nokogiri Gem
   sudo dnf -y install gcc-c++                            # For event-machine Gem
+  sudo dnf -y install openssl-devel                      # For rubygems
   ```
 
-#### CentOS 7
-
-* Enable EPEL
+* Enable Memcached
 
   ```bash
-  sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  sudo systemctl enable memcached
+  sudo systemctl start memcached
   ```
 
-* Install packages
-
-  ```bash
-  sudo yum -y install git-all memcached postgresql-devel postgresql-server \
-  libxml2-devel libxslt-devel gcc-c++ patch
-  ```
-* Initialize postgresql and enable services
+* Configure PostgreSQL
 
   ```bash
+  sudo passwd postgres <new_password>
   sudo postgresql-setup initdb
-  sudo su postgres -c "echo local all all trust > /var/lib/pgsql/data/pg_hba.conf"
-  sudo systemctl start memcached
-  sudo systemctl enable memcached
-  sudo systemctl start postgresql
+  sudo sed -i.bak 's/\(^local\s*\w*\s*\w*\s*\)\(peer$\)/\1trust/' /var/lib/pgsql/data/pg_hba.conf
```

Comment:
`md5` is probably a good idea.  I just consolidated what was there, I'll update it.